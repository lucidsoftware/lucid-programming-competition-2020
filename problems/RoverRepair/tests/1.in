7
8
3
14
1
6
empty
10
