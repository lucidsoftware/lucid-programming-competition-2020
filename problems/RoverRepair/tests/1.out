Swap nodes 10 and 14
